lib/sanctify/matcher.rb

```diff
@@ -0,0 +1,27 @@
+module Sanctify
+  class Matcher
+    attr_reader :description, :regex, :config
+    def initialize(description, regex, config)
+      @description = description
+      @regex = regex
+      @config = config
+    end
+
+    def id
+      description.downcase.gsub(' ', '_')
```

Comment:
This feels like meta-programming, and can make the code hard to follow. Instead, I'd suggest that each matcher define a unique ID for itself **and** a human-readable description, and not couple them like this. As a bonus, it would simplify the logic for determining whether a matcher should be disabled, and allow it to be hoisted to the caller initializing a Matcher object.

Reply:
Yeah I thought about this before I committed. I did add the list of acceptable ids to the README but as this grows, we'll want to make that more explicit. Good call.